src/pybind/rados/rados.pyx

```diff
@@ -298,6 +298,7 @@ cdef extern from "rados/librados.h" nogil:
     void rados_write_op_remove(rados_write_op_t write_op)
     void rados_write_op_truncate(rados_write_op_t write_op, uint64_t offset)
     void rados_write_op_zero(rados_write_op_t write_op, uint64_t offset, uint64_t len)
+    void rados_write_op_assert_exists(rados_write_op_t write_op)
```

Comment:
@zjcmszh could you order the declaration alphabetically?

Reply:
I'll do it right away.

Reply:
thanks

Reply:
thanks yoursuggestion , updated.

Reply:
thanks your suggestion , updated.